06-files-input-and-output/csv_split.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2016 Ami . All rights reserved
+import argparse
+import csv
+import os
+import sys
+
+def get_arguments():
+    current_directory = os.getcwd()
+    parser = argparse.ArgumentParser(description="Split CSV by every x rows")
+
+    parser.add_argument("-i", action="store", dest="input_file", help="CSV input file")
+    parser.add_argument("-o", action="store", dest="output_path",
```

Comment:
Break the line such that each argument is on a new line. Note that the add_argument takes both positional arguments and named arguments (the flags are positional and don't have a name). I this case, only the named arguments should appear in the form of `key=value` but each of the arguments should appear on a separate line.